inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -80,8 +89,19 @@ void FrontEnd::parseInterp(const Model& model, const ie::CNNLayerPtr& layer, con
     VPU_THROW_UNLESS(outputs.size() == 1,
                      "Interp stage with name {} must have only 1 output, "
                      "actually provided {}", layer->name, outputs.size());
+    ie::details::CaselessEq<std::string> cmp;
+    const auto coord = layer->GetParamAsString(g_coordinate_transformation_mode, g_half_pixel);
+    const auto interpMode = layer->GetParamAsString(g_mode, g_linear);
+    InterpolateCoordTransMode coordinateTransMode = InterpolateCoordTransMode::HalfPixel;
```

Comment:
Do we really need to declare these variables here with initialization?

Reply:
Yes, as in assigning it to the default values.